vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -571,8 +561,8 @@ def _compute_cos_sin_cache(
         inv_freq = self._compute_inv_freq(rescale_factors)
         t = torch.arange(max_position_embeddings, dtype=torch.float)
         freqs = torch.einsum("i,j -> ij", t, inv_freq)
-        cos = freqs.cos() * mscale * self.scaling_factor
-        sin = freqs.sin() * mscale * self.scaling_factor
+        cos = freqs.cos() * mscale
+        sin = freqs.sin() * mscale
```

Comment:
I don't think we can accept this change as it will affect other models or context length extensions, is there a way you could add a condition for your case to set scaling_factor to 1?